### PR TITLE
Minor changes to cancer type page based on feedback

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: src/main/webapp/app/oncokb-frontend-commons/src/components/LongText.tsx [hunk 1]
```diff
@@ -2,27 +2,35 @@ import * as React from 'react';
 import { useState } from 'react';
 import SummaryWithRefs from './SummaryWithRefs';
 import styles from '../../../index.module.scss';
+import classnames from 'classnames';
+import showHideText from 'app/shared/texts/ShowHideText';
```
Comment (calvinlu3) [inline src/main/webapp/app/oncokb-frontend-commons/src/components/LongText.tsx:6]: Not used

File: src/main/webapp/app/oncokb-frontend-commons/src/components/LongText.tsx [hunk 2]
```diff
@@ -2,27 +2,35 @@ import * as React from 'react';
...
 
 const shortenTextByCharacters = (text: string, cutoff: number) => {
```
Comment (calvinlu3) [inline src/main/webapp/app/oncokb-frontend-commons/src/components/LongText.tsx:8]: Seems like this can be reused elsewhere. Maybe move to a utils file?

**Review by calvinlu3 — APPROVED** [review]
LGTM
Do we also want to make the book icon in description column to have the style cursor: pointer?

**Comment by zhx828** [comment]
> LGTM Do we also want to make the book icon in description column to have the style cursor: pointer?

@calvinlu3 it's not clickable, I would not.

**Pull request merged**